Task-5/main.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"FrontEndToDB/myconn"
+	myQueries "FrontEndToDB/mydb"
+	"fmt"
+	"html/template"
+
+	"net/http"
+)
+
+func main() {
+
+	// Making connection to postgres server
+	db := myconn.ConnectToDB(5555, "postgres", "1234", "postgres")
```

Comment:
Use Enviroment variable for the database creds, database details.

@Tridipchavda

Reply:
@Tridipchavda 

don't push .env files on Github
for that use .gitignore for future reference.